css_tasks/task9/index.html

```diff
@@ -0,0 +1,378 @@
+      <nav class="header__navbar">
+        <ul class="header-navbar">
+          <li class="header-navbar__item"><a href="#">Features</a></li>
+          <li class="header-navbar__item"><a href="#">About</a></li>
+          <li class="header-navbar__item"><a href="#">Pricing</a></li>
+          <li class="header-navbar__item"><a href="#">Reviews</a></li>
+          <li class="header-navbar__item"><a href="#">Contact</a></li>
+        </ul>
+      </nav>
+   
+    <span class="header__mobile-icon"><svg aria-hidden="true" focusable="false" data-prefix="fas" data-icon="bars" class="svg-inline--fa fa-bars fa-w-14" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><path fill="#fff" d="M16 132h416c8.837 0 16-7.163 16-16V76c0-8.837-7.163-16-16-16H16C7.163 60 0 67.163 0 76v40c0 8.837 7.163 16 16 16zm0 160h416c8.837 0 16-7.163 16-16v-40c0-8.837-7.163-16-16-16H16c-8.837 0-16 7.163-16 16v40c0 8.837 7.163 16 16 16zm0 160h416c8.837 0 16-7.163 16-16v-40c0-8.837-7.163-16-16-16H16c-8.837 0-16 7.163-16 16v40c0 8.837 7.163 16 16 16z"></path></svg></span>
+    </header>
+    <section class="jumbotron">
+      <picture>
+        <source media="(max-width:678px)" srcset="assets/image-min.png" />
+        <source
+          media="(min-width:679px) and (max-width:1024px)"
+          srcset="assets/image@2x-min.png"
+        />
+        <img
+          src="assets/image@3x-min.png"
+          alt="Buildings"
+          class="jumbotron__image"
+        />
+      </picture>
+      <div class="container jumbotron__content">
+        <div class="jumbotron-content__left-column">
+          <header class="jumbotron-left-column__header">
+            <h1 class="jumbotron-header__title">
+              <span>ax</span><span>it</span>
+            </h1>
+            <p class="jumbotron-header__subtitle">
+              modern axure template <br />for beautiful prototypes
+            </p>
+          </header>
+          <div class="jumbotron-left-column__separator"></div>
+          <p class="jumbotron-left-column__text">
+            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aenean
+            euismod bibendum laoreet. Proin gravida dolor sit amet lacus
+            accumsan et viverra justo commodo.
+          </p>
+          <button class="jumbotron-left-column__button">Download</button>
+        </div>
+        <form action="" class="jumbotron__form">
+          <h4 class="jumbotron-form__title section__title" >try your <span>free</span> trial today</h4>
+          <input class="jumbotron-form__input" type="text" name="name" placeholder="Name">
+          <input class="jumbotron-form__input" type="email" name="email" placeholder="Email">
+          <input class="jumbotron-form__input" type="password" name="password" placeholder="Password">
+          <button class="jumbotron-form__button">get started</button>
+        </form>
+      </div>
+      <section class="jumbotron__social">
```

Comment:
it is not part of jumbotron

Reply:
really?

Reply:
probably yes, looks just like separate section